internal/cli/atlas/performanceadvisor/slowquerylogs/list.go

```diff
@@ -28,7 +28,7 @@ import (
 	atlasv2 "go.mongodb.org/atlas-sdk/admin"
 )
 
-const listTemplate = `NAMESPACE	LINE{{range .SlowQuery}}
+const listTemplate = `NAMESPACE	LINE{{range .SlowQueries}}
```

Comment:
Could you add a test to check that the template is correct? You can use this PR as an example https://github.com/mongodb/mongodb-atlas-cli/pull/1471

Reply:
Awesome suggestion! We already have unit test for each command but that does use raw json.
I'm investigating if we can do this for all commands with little effort track those issues in the future.

Reply:
there is a ticket in the backlog to add this check to all the unit tests